### PR TITLE
Bumped dependency SDK versions

### PR DESCRIPTION
**Pull request by vincethecoder**
* Bumped vision-ar's pod and carthage dependencies to latest releases.

/cc @mapbox/vision-ios @atripathi-mb @okoriep @akitchen @andrewychen

### CODE REVIEW

**Comment by atripathi-mb** [comment]
cc @vladzima @chezzdev @deniskoronchik

File: MapboxVision.podspec [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "MapboxVision"
-  s.version      = "0.3.0"
+  s.version      = "0.3.1"
```
Comment (chezzdev) [inline MapboxVision.podspec]: There's no need to update the version. It's a part of the release process.

File: MapboxVisionAR.podspec [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "MapboxVisionAR"
-  s.version      = "0.3.0"
+  s.version      = "0.3.1"
```
Comment (chezzdev) [inline MapboxVisionAR.podspec]: Same here. Version bump is a separate process.

File: MapboxVisionAR.podspec [hunk 3]
```diff
@@ -24,8 +24,8 @@ Pod::Spec.new do |s|
 
   s.swift_version = '4.1'
 
-  s.dependency "MapboxVisionNativeAll/AR", "~> 0.3.0"
+  #s.dependency "MapboxVisionNativeAll/AR", "~> 0.3.0"
```
Comment (chezzdev) [inline MapboxVisionAR.podspec]: Why was the string commented out?

Comment (vincethecoder) [reply MapboxVisionAR.podspec]: That's an accident. thanks for catching that.

File: MapboxVision.podspec (hunk 1 above)
Comment (vincethecoder) [reply MapboxVision.podspec]: Got it.

File: MapboxVisionAR.podspec (hunk 2 above)
Comment (vincethecoder) [reply MapboxVisionAR.podspec]: Okay. I will revert this.

**Comment by vincethecoder** [comment]
@chezzdev I will be closing this PR in favor of branching off the `dev` branch and bumping up the dependencies accordingly. Thanks for the feedback.

**Pull request closed without merging**